doc/source/_static/numpy.css

```
@@ -32,3 +31,24 @@ h3 {
   color: #013243; /* warm black */
   letter-spacing: -.03em;
 }
+
+.intro-card {
+  padding: 20px 10px 20px 10px;
+  margin: 10px;
+}
```

For the no-border on the gray.
```suggestion
.intro-card {
  padding: 20px 0px 20px 0px;
  margin: 10px;
}
```